### ISSUE TITLE
Migrate specs to https://github.com/Azure/azure-rest-api-specs

### ISSUE THREAD

**Issue by nschonni** [MicrosoftDocs/vsts-rest-api-specs#201]
I know the "Azure" was just a marketing thing, but that repo holds the other Azure Swagger files. It also has validation tools and generates out SDKs automatically for multiple languages.
They do also get published as API docs through the different language SDK sites like https://github.com/MicrosoftDocs/azure-docs-sdk-node

**Comment by nmiodice** [comment]
Agreed, this is an ideal path forward. Work needs to be done to make these OpenAPI compliant (https://github.com/MicrosoftDocs/vsts-rest-api-specs/issues/161) but if it happens, then it opens up the door to generating clients in many languages to interact w/ AZDO.
